cardano-testnet/src/Testnet/Conf.hs

```diff
@@ -39,6 +40,7 @@ mkConf tempAbsPath maybeMagic = do
   tempBaseAbsPath <- H.noteShow $ FP.takeDirectory tempAbsPath
   tempRelPath <- H.noteShow $ FP.makeRelative tempBaseAbsPath tempAbsPath
   base <- H.noteShowM H.getProjectBase
+  configurationTemplate <- H.noteShow $ base </> "configuration/defaults/byron-mainnet/configuration.yaml"
```

Comment:
I don't understand how this is now configurable. We are still defaulting to "configuration/defaults/byron-mainnet/configuration.yaml". `mkConf` should have an additional `ConfigYamlFilePath` parameter.

Reply:
The long term goal will be to have an additional parser at the top level that accepts a configuration yaml file but the change I described above is good for now.

Reply:
This is what I had in mind: https://github.com/input-output-hk/plutus-apps/pull/383